### PR TITLE
Catch `ax` keyword argument in `scatter3d` and add FAQ on subplots with 3d figures

### LINKED ISSUE #116: Cannot use plopp.scatter3d with matplotlib subplots
Trying to use `plopp.scatter3d` with matplotlib subplots gives an error: "unexpected keyword argument 'ax'".

Here is a code example to reproduce the error

```
time = np.linspace(0, 10, 50)
x = np.cos(time)
y = np.sin(time)

da = sc.DataArray(
    data=sc.array(dims=['row'], values=time),
    coords={
        'x': sc.array(dims=['row'], unit='m', values=x),
        'y': sc.array(dims=['row'], unit='m', values=y),
        'time': sc.array(dims=['row'], unit='s', values=time)
    })

fig, ax = plt.subplots(1, 2, figsize=(10, 10))

p1 = pp.plot(da, ax=ax[0])
p2 = pp.scatter3d(da, x='x', y='y', z='time', pixel_size=0.2, ax=ax[1])

fig.tight_layout()
```

### PR DESCRIPTION
**Pull request by nvaytet**
Fixes #116

### CODE REVIEW

File: src/plopp/functions/scatter3d.py [hunk 1]
```diff
@@ -66,6 +66,13 @@ def scatter3d(da: sc.DataArray,
     from ..functions import figure3d
     from ..widgets import Box, ToggleTool, TriCutTool
 
+    if 'ax' in kwargs:
+        raise ValueError(
```
[the remark this answers is not in the document]
Comment (nvaytet) [reply src/plopp/functions/scatter3d.py:70]: I looked around and concensus on Stack Overflow was to raise `ValueError`. `InvalidArgument` seems to be a `hypothesis` thing?

[the remark this answers is not in the document]
Comment (SimonHeybrock) [reply src/plopp/functions/scatter3d.py:70]: Actually I get `TypeError`: `TypeError: f() got an unexpected keyword argument 'ax'` for a dummy function `f`.

File: docs/examples/faq.ipynb [hunk 2]
```diff
@@ -0,0 +1,60 @@
+{
```
Comment (SimonHeybrock) [inline docs/examples/faq.ipynb:1]: Do you plan to make an FAQ page specific for the example (and thus other sections)? Or should it be in a more generic location?

Comment (nvaytet) [reply docs/examples/faq.ipynb:1]: It's supposed to be generic, not sure where to put it.
A section of its own? Change the `Reference` section to something else and put it next to the API reference?

Comment (SimonHeybrock) [reply docs/examples/faq.ipynb:1]: Scipp has it in the `Getting Started` section: https://scipp.github.io/getting-started/faq.html

**Review by SimonHeybrock — APPROVED** [review]

**Pull request merged**